tests/helpers/perf_results_helper.rb

```diff
@@ -66,8 +66,7 @@ module PerfResultsHelper
                            "catalog",
                            "report"].freeze
 
-  # TODO: update to 'puppet-metrics-collector' (SLV-589)
-  PUPPET_METRICS_COLLECTOR_DIR_NAME = "puppet_metrics_collector"
+  PUPPET_METRICS_COLLECTOR_DIR_NAME = "puppet-metrics-collector"
```

Comment:
Does the [spec test](https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/spec/tests/helpers/perf_results_helper_spec.rb#L399) need to be updated?

Reply:
No, because I use `puppet_metrics_dir_name = PerfResultsHelper::PUPPET_METRICS_COLLECTOR_DIR_NAME` in the spec.